File: Leetcode Problems/twoSum.cpp
#include <stdio.h>
#include <vector>
using namespace std;
class Solution{
    public:
        vector<int> twoSum(vector<int>& nums, int target){
            vector<int> v;
            bool flag = false;
            for(int i=0;i<nums.size()-1;i++)
            {
                for(int j=i+1;j<nums.size();j++)
                {
                    if(nums[i]+nums[j]==target)
                    {
                        v.push_back(i);
                        v.push_back(j);
                        flag = true;
                        break;
                    }
                }
                if(flag == true)
                {
                    break;
                }
            }
            return v;
        }
};

File: Leetcode Problems/TwoSum.java
import java.util.*;

class Solution {
    public int[] twoSum(int[] nums, int target) {
        int size = nums.length;
        int tempOne;
        int tempTwo;
        int indexOne = 0;
        int indexTwo = 0;
        for(int i = 0; i<size; i++){
            tempOne = nums[i];
            for(int j = 0; j<size; j++){
                if(j==i){
                    continue;
                }
                else{
                    tempTwo = nums[j];
                    if(tempTwo+tempOne==target){
                        indexOne = i;
                        indexTwo = j;
                    }
                }
            }
        }
        int [] arr = {indexOne, indexTwo};
        return arr;
    }
}
